cityofaustin/atd-moped
# Add new funds to moped_funds

## Associated issues

https://github.com/cityofaustin/atd-data-tech/issues/9676

We previously had the `fund_id` as an int, but one of the funds to be added is 820B, so I changed the column to text. 

## Testing
**URL to test:** 
~~Database changes, so you need to run locally or moped test~~

https://9676-add-fund-sources--atd-moped-main.netlify.app/moped/projects


**Steps to test:**

On the funding tab, add a new funding source and check for the new funds in the dropdown list. 

![Screen Shot 2022-07-26 at 5 26 18 PM](https://user-images.githubusercontent.com/12474808/181123010-123336a0-d544-4e45-a1e5-327b07066c52.png)

---
#### Ship list
